src/promptflow-evals/promptflow/evals/synthetic/simulator/_model_tools/cogservices_captioning.py

```diff
@@ -0,0 +1,121 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+import json
+import urllib
+from typing import Any, Dict, Optional, Union
+
+from aiohttp import ClientTimeout  # pylint: disable=networking-import-outside-azure-core-transport
+from aiohttp_retry import RetryClient  # pylint: disable=networking-import-outside-azure-core-transport
+
+from promptflow.evals.synthetic.simulator._model_tools.identity_manager import KeyVaultAPITokenManager

```

Comment:
Lets check if this is needsed if not lets remove dependency on azure-keyvault-secrets package

Reply:
removed :)